src/main/java/school/lemon/changerequest/java/introduction/hw2/Task2.java

```diff
@@ -3,6 +3,14 @@
 
 public class Task2 {
     public static String firstTwo(String s) {
-        return "";
+
+        if (s == null) {
+            return null;
+        } else if (s.length() < 2) {
+            return s;
+        } else {
+            return s.substring(0, 2);
+        }
```

Comment:
General comment: there is no need for additional `else` clauses when you do `return`, cause there is no way to continue if you're returning from function.

Also in this task there is no need for second `else-if` clause, as it could be merged into first `if` clause condition:
```java
if (s == null || s.length <=2){
return s;
}
```